testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java

```diff
@@ -0,0 +1,4 @@
+package org.jboss.as.test.integration.common;
```

Comment:
Would you mind moving also the license header to the common file? I would say it is not mandatory for merging, but we should keep the pattern we already have.